zebra/zebra_rib.c

```diff
@@ -2070,6 +2076,25 @@ static void rib_process_result(struct zebra_dplane_ctx *ctx)
 	}
 
 	if (op == DPLANE_OP_ROUTE_INSTALL || op == DPLANE_OP_ROUTE_UPDATE) {
+		if (re && re->nhe_id >= ZEBRA_NHG_PROTO_LOWER) {
+			nhe = zebra_nhg_lookup_id(re->nhe_id);
+			if (nhe) {
+				srcdest_rnode_prefixes(rn, &p, &src_p);
+				prefix_copy(&nhg_p.prefix, p);
+				if (src_p)
+					prefix_copy(&nhg_p.src_prefix, src_p);
+				nhg_p.table_id = dplane_ctx_get_table(ctx);
+				nhg_p.afi = dplane_ctx_get_afi(ctx);
+				nhg_p.safi = dplane_ctx_get_safi(ctx);
+				nhg_p.vrf_id = dplane_ctx_get_vrf(ctx);
+				nhg_p.back_ptr = nhe;
```

Comment:
TBH, back_ptr is not the best name here. Why not just `nhe`?

Reply:
done